From f0c81ce4746ba676944a5f25efbde9de5054c341 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 24 Feb 2021 10:52:30 +0800
Subject: [PATCH] drm/rockchip: use DRM_FB_HELPER_DEFAULT_OPS for fb_ops

Restore upstream commit 72b6bcb1a2a4 ("drm/rockchip: use DRM_FB_HELPER_DEFAULT_OPS for fb_ops")

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I523336e7d9374ed535c558e5ab49d9b8f89cfb8f
---
 drivers/gpu/drm/rockchip/rockchip_drm_fbdev.c | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_fbdev.c b/drivers/gpu/drm/rockchip/rockchip_drm_fbdev.c
index e349106bdaa6..67a21fd1609c 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_fbdev.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_fbdev.c
@@ -52,14 +52,7 @@ static struct dma_buf *rockchip_fbdev_get_dma_buf(struct fb_info *info)
 
 static struct fb_ops rockchip_drm_fbdev_ops = {
 	.owner		= THIS_MODULE,
-	.fb_check_var	= drm_fb_helper_check_var,
-	.fb_set_par	= drm_fb_helper_set_par,
-	.fb_setcmap	= drm_fb_helper_setcmap,
-	.fb_blank	= drm_fb_helper_blank,
-	.fb_pan_display	= drm_fb_helper_pan_display,
-	.fb_debug_enter = drm_fb_helper_debug_enter,
-	.fb_debug_leave = drm_fb_helper_debug_leave,
-	.fb_ioctl	= drm_fb_helper_ioctl,
+	DRM_FB_HELPER_DEFAULT_OPS,
 	.fb_fillrect	= drm_fb_helper_cfb_fillrect,
 	.fb_copyarea	= drm_fb_helper_cfb_copyarea,
 	.fb_imageblit	= drm_fb_helper_cfb_imageblit,
-- 
2.35.3

